### PR TITLE
Spectre (Update)

### PR DESCRIPTION
**Pull request by ThatOneGoblin25**
## About the PR
- Reworked all rooms
- Wiring rework
- Atmos rework
- Decal rework
- Added atmos piping to artifact chambers
- Added robotics
- Added plasma, carbon and empty canisters
- Added a power drill as a RD item
- Added a science techfab
- Added a television to bar
- Added Donut Vend
- Added a Genedrobe with a secret wall below it
- Replaced most consoles with tabletop variants
- Removed blast doors over artifact chamber entrance
- Removed medical doctor's wall locker
- Removed some of the artifact containers
- Trimmed the amount of lights
- Tweaked bar, booze-o-mat instead of soda dispenser
- Tweaked price, from 195000 to 1850000
- Renamed from NSS Spectre to NR Spectre
- New description.

## Why / Balance
Updating to new mapping standards, also only large ship to be missing a mothership console.

## Media
Screenshot 2024-03-23 003631
Screenshot 2024-03-23 003618
Screenshot 2024-03-23 003554
Screenshot 2024-03-23 003533

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- Tweak: Updated the Spectre to current mapping guidelines with additional features.

### CODE REVIEW

**Comment by ThatOneGoblin25** [comment]
@Wolfhauler Letting you know I've proposed some changes to your ship.

**Comment by ThatOneGoblin25** [comment]
Map checker failed because of the telecomms having Cargo, Engineering, Science and Service encryption keys.

**Comment by Wolfhauler** [comment]
o7

**Comment by Wolfhauler** [comment]
crematorium was denied last time by the by

**Comment by MagnusCrowe** [comment]
I don't understand why a research ship would have a mothership console that allows you to get Pioneer, Prospector, Chisel, Bocadillo, PTS, Construct.  It makes no sense to me.

Why the Smile AI?

**Comment by ThatOneGoblin25** [comment]
> I don't understand why a research ship would have a mothership console that allows you to get Pioneer, Prospector, Chisel, Bocadillo, PTS, Construct. It makes no sense to me.
Drawing from its description: "A large research mothership designed to be flown next to a small fleet of other ships including salvage and food services." It should be able to provide these ships conveniently to the crew on board. The Crescent mothership console provides all the ships I listed and more. 
There wasn't much point I saw in adding additional sci ships like the Sparrow or Investigator, since they do the same job as the main ship and we want to keep activity on there. I did have a vague idea to make a unique ship, a solar-powered shuttle that only had an artifact scanner and analysis console for this console but I realised it would detract from what the Spectre could offer.

As for the Smile AI, it's something unique that draws parallels to sci labs on stations. The largest sci-ship on the market should draw similarities to such sci labs. But it can go as the ghost role isn't whitelisted, you can't DO anything other than talk or suicide but if you believe it unnecessary I can remove it.

**Comment by MagnusCrowe** [comment]
Nah, keep Smile.  Makes more sense for it to be on the Spectre than the Wasp anyway

**Comment by dvir001** [comment]
> Nah, keep Smile. Makes more sense for it to be on the Spectre than the Wasp anyway

Fucker worth 3k dead

**Comment by MagnusCrowe** [comment]
> > Nah, keep Smile. Makes more sense for it to be on the Spectre than the Wasp anyway
> 
> Fucker worth 3k dead

I feel like that is a separate issue that should be addressed...why is it worth that much?

**Comment by ThatOneGoblin25** [comment]
I really feel like the Spectre deserves a mothership console still. Prehaps nerfed slightly in its options. It's really a sci base away from station and a substitute for Anom Labs with the Anomaly generator. As for docks, even with only two I think it can be managed by any crew, it is a tiny obstacle managing the docks.

Also map checker fails for the encryption keys in the telecomms machine, it's something the original creator added, should I remove it @dvir001 ?

**Comment by ThatOneGoblin25** [comment]
Ignore the commit spaghetti
Updated PR screenshots
- Removed the mothership console and textures
- Added a donut vendor, for a large crew have *no* food feels a bit unfair, especially since food services aren't guaranteed. Half the donuts are poisonous to felinids and vulps which is hilarious.
- Added a borg recharger
- Made gyros look better. This requires LV cables to run under reinforced walls but its rather minor, and is only used to power two gyros on the tips of the ship. I felt this was necessary to preserve the sleek ship design with minimal sacrifices.

**Review by Cheackraze — APPROVED** [review]
Im not personally opposed to the _idea_ of having the mothership console here, the problem I have is that it should be a very narrow, restricted 'themed' list of mini science drone ships, instead of just a generic list of food/general RP ships. The rest of the changes here are fine

**Comment by dvir001** [comment]
No food vendors on ships, having the restock option make it infinite food machine that remove the need to leave it.

File: Resources/Prototypes/_NF/Shipyard/spectre.yml [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
+  description: A large, dashing, but dated vessel with a pure focus on research and development. It is capable of generating anomalies.
+  price: 185000
   category: Large
   group: Civilian
   shuttlePath: /Maps/_NF/Shuttles/spectre.yml
 
 - type: gameMap
   id: Spectre
-  mapName: 'NSSSpectre'
+  mapName: 'Spectre'
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Shipyard/spectre.yml:22]: Please use the same map name format as all other ships

Comment (ThatOneGoblin25) [reply Resources/Prototypes/_NF/Shipyard/spectre.yml]: Done

**Comment by ThatOneGoblin25** [comment]
Removed Donut Vendor
Removed Drill
Removed Two Canisters
Removed slips

**Comment by ThatOneGoblin25** [comment]
> the problem I have is that it should be a very narrow, restricted 'themed' list of mini science drone ships

I am willing to do this. Please hold off on merging.

File: Resources/Prototypes/_NF/Entities/Structures/Machines/Computers/mothership-computers.yml [hunk 2]
```diff
@@ -100,6 +100,27 @@
     - Mission
     - Honker
 
+# The spectre console
+- type: entity
+  id: SpectreMothershipComputer
+  name: Spectre mothership console
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Entities/Structures/Machines/Computers/mothership-computers.yml:106]: Lower case in names

File: Resources/Prototypes/_NF/Shipyard/ghost.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# Discord: ???
+
+# Maintainer Info
+# GitHub: ???
+# Discord: ???
+
+# Shuttle Notes:
+#
+- type: vessel
+  id: PTS
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Shipyard/ghost.yml:12]: You cannot just copy paste an existing entity

**Review by dvir001 — CHANGES_REQUESTED** [review]

**Comment by MagnusCrowe** [comment]
I cannot actually build the code in this PR so I can't review it at this time.  Does it build for you?

**Comment by ThatOneGoblin25** [comment]
Added the NX Ghost shuttle for the Spectre Mothership Console.
Screenshot 2024-04-02 161034

**Comment by ThatOneGoblin25** [comment]
I am like 90% sure i screwed up my own fork and if I reopened this on a new branch this wouldn't happen :P

**Comment by ThatOneGoblin25** [comment]
Let's try opening a new branch.

**Pull request closed without merging**